adapters.go

```diff
@@ -107,6 +142,11 @@ func AdaptCosmosTxsToEthTxs(cosmosTxs bfttypes.Txs) (ethtypes.Transactions, erro
 	if len(ethTxsBytes) == 0 {
 		return nil, errL1AttributesNotFound
 	}
+	return ethTxsBytes, nil
+}
+
+func mergeEthAndCosmosTxs(ethTxsBytes, txsBytes [][]byte) (ethtypes.Transactions, error) {
+	txs := make(ethtypes.Transactions, 0, len(ethTxsBytes)+len(txsBytes))
```

Comment:
The conversion from lines 149-159 could be done in `getEthTxsBytes` - fits naturally there.

Then this helper could probably be removed, since inlining the rest is pretty small and readable.